### PR TITLE
Migration instructions

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,45 @@ SiteSetting.sso_overrides_name = true
 SiteSetting.sso_overrides_email= true
 ```
 
+## Import from v5
+
+Loading the v5 database
```
Comment (asaunier) [inline README.md:43]: Perhaps you could add the triple backquotes to make the command lines appear as code?

File: README.md [hunk 2]
```diff
@@ -38,3 +38,45 @@ SiteSetting.sso_overrides_name = true
...
+Add missing columns:
+ALTER TABLE punbb_topics ADD COLUMN first_post_id integer default 0;
+UPDATE punbb_topics SET first_post_id = (select MIN(p.id) from punbb_posts as p where punbb_topics.id = p.topic_id);
+
+A discourse installation in developer mode is required:
+https://github.com/discourse/discourse/blob/master/docs/DEVELOPER-ADVANCED.md
+
+git clone https://github.com/discourse/discourse
+cd discourse
+(gem install bundle --user-install)
```
Comment (asaunier) [inline README.md]: What does this parenthesed mean?

File: README.md [hunk 3]
```diff
@@ -38,3 +38,45 @@ SiteSetting.sso_overrides_name = true
...
+PGHOSTADDR=127.0.0.1 PGUSER=discourse_import  PGPASSWORD=discourse_import bundle exec rake db:create db:migrate
+
+
+
+Migration:
+date; DATABASE_URL=postgres://discourse_import:discourse_import@127.0.0.1/discourse_development bundle exec ruby  script/import_scripts/punbb.rb; date
+
+## Testing (with standalone, non sso instance)
+
+Set 'c2cc2cc2' password to everyone:
```
Comment (asaunier) [inline README.md]: Is it actually `c2cc2cc2` (OK for me, just want to make sure that's not a typo)?

**Comment by asaunier** [comment]
A few minor comments else looks good.

Do we still need c2corg's fork of Discourse?
https://github.com/c2corg/discourse

**Comment by gberaudo** [comment]
Sure we need it. Updated the README.md with your comments.

**Comment by gberaudo** [comment]
Thanks for the comments @asaunier.

**Pull request merged**